ci/emscripten.sh

```diff
@@ -2,8 +2,6 @@
 
 set -eux
 
-# Note: keep in sync with:
-# https://github.com/rust-lang/rust/blob/master/src/ci/docker/scripts/emscripten.sh
```

Comment:
Is there something else we should be referencing for version to test with? Maybe the platform docs.

Reply:
I've updated this link to https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md#requirements instead.